sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_phone_numbers_client.py

```diff
@@ -189,13 +190,19 @@ def begin_update_phone_number_capabilities(
             for LRO operations if no Retry-After header is present.
         :rtype: ~azure.core.polling.LROPoller[~azure.communication.phonenumbers.models.PurchasedPhoneNumber]
         """
-        return self._phone_number_client.phone_numbers.begin_update_capabilities(
+        poller = self._phone_number_client.phone_numbers.begin_update_capabilities(
             phone_number,
             calling=calling,
             sms=sms,
             **kwargs
         )
 
+        result_properties = poller.result().additional_properties
+        if 'status' in result_properties and result_properties['status'] == 'failed':
```

Comment:
Is this check case sensitive?

Reply:
Yes, this check is case sensitive.

Reply:
Discussed this offline. The status value isn't always lower case, so this check should handle that.